sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/TransientIOErrorsRetryingIteratorSpec.scala

```diff
@@ -81,7 +81,8 @@ class TransientIOErrorsRetryingIteratorSpec extends UnitSpec with BasicLoggingTr
       cosmosRowConverter.fromObjectNodeToRow(
         ItemsTable.defaultSchemaForInferenceDisabled,
         node,
-        SchemaConversionModes.Strict))
+        SchemaConversionModes.Strict),
+      None)
```

Comment:
nit: indent

Reply:
Was actually correct - but added linebreak to make it easier to read in next iteration.